drivers/SmartThings/philips-hue/src/handlers/lifecycle_handlers.lua

```diff
@@ -0,0 +1,120 @@
+local log = require "log"
```

Comment:
I typically follow the practice of having an init.lua file in the directory rather than a lua file outside of the directory where there are submodules/other lua files. I think we have followed this for almost everything in our lua libs (except cosock) and drivers. Its not a huge deal, but just sort of weird IMO to see directories without an init.lua file in them. Was there a reason you did it the way you did?

Reply:
Yeah, I wanted to minimize the occurrences of [this](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1334#discussion_r1578241412) after Steven's comment on the prior PR.